### ISSUE TITLE
[BUG] java.nio.channels.ClosedChannelException   &  writeAddress(..) failed

### ISSUE THREAD

**Issue by wls981** [apache/shenyu#5624]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I'm experiencing some issues that are hard to reproduce and solve, such as the following exceptions, which occur 1000-2000 times a day in production. The exceptions are very random and difficult to reproduce directly. I need help!
![image](https://github.com/user-attachments/assets/881cddd8-0cb3-471b-99b5-3c47427a225f)
![image](https://github.com/user-attachments/assets/0920bc7c-3331-4c0d-842b-e0a437f032c1)

### Steps To Reproduce

The problem occurs very randomly and is difficult to reproduce directly.

### Environment

```markdown
ShenYu version(s):2.4.3
```

**Comment by Aias00** [comment]
u have some custom plugin?

**Comment by wls981** [comment]
Yes, there are several, such as plugins that support full-link grayscale publishing, and plugins that support sse responses.
